lib/cpp/src/thrift/TApplicationException.h

```diff
@@ -21,6 +21,7 @@
 #define _THRIFT_TAPPLICATIONEXCEPTION_H_ 1
 
 #include <thrift/Thrift.h>
+#include <stdint.h>
```

Comment:
This is a little more complicated than I anticipated.  It looks like when compiling plugin_types.cpp we end up including TApplicationException through some code path that either undefines HAVE_STDINT_H (which is defined in windows/config.h, from thrift/Thrift.h) or something else is happening.  Still working on it.